src/flux/task-step.coffee

```diff
@@ -5,13 +5,17 @@ flux = require 'flux-react'
 
 TaskStepConfig =
 
+  _loaded: (obj, id) ->
+    _.extend(@_local[id], obj)
```

Comment:
The reason for this is that it may come from either @_local or obj.  I've updated it to explicitly set it for now, with some additional logic.  Hopefully we can have the backend send use the task_id directly eventually.